build_tools/github_actions/configure_ci.py

```diff
@@ -246,44 +300,124 @@ def modifies_included_path(base_ref: str) -> bool:
     return any(not skip_path(p) for p in get_modified_paths(base_ref))
 
 
-def should_run_ci(is_pr: bool, trailers: Mapping[str, str]) -> bool:
+def get_runner_env(trailers: Mapping[str, str]) -> str:
+    runner_env = trailers.get(Trailer.RUNNER_ENV)
+    if runner_env is None:
+        print(
+            f"Using '{RUNNER_ENV_DEFAULT}' runners because"
+            f" '{Trailer.RUNNER_ENV}' not found in {trailers}"
+        )
+        runner_env = RUNNER_ENV_DEFAULT
+    else:
+        print(
+            f"Using runner environment '{runner_env}' from PR description" f" trailers"
+        )
+    return runner_env
+
+
+def parse_jobs_trailer(
+    trailers: Mapping[str, str], key: str, all_jobs: Set[str]
+) -> Set[str]:
+    jobs_text = trailers.get(key)
+    if jobs_text is None:
+        return set()
+    jobs = set(jobs_text.split(","))
```

Comment:
Should we strip whitespace from the job after split?

Reply:
Good call, done.